Delete Risk Meters/deleteriskmeters.py

```diff
@@ -0,0 +1,28 @@
+import requests
+import pandas as pd
+import sys
+
+token = sys.argv[1]
+file = sys.argv[2]
+
+headers = {"Accept": "application/json", "X-Risk-Token": token}
+
+base_url = "https://api.kennasecurity.com"
```

Comment:
Please add a comment here that the `base_url` might have to change depending on your configuration.